dev/ci/ci-elpi.sh

```diff
@@ -6,3 +6,7 @@ ci_dir="$(dirname "$0")"
 git_download elpi
 
 ( cd "${CI_BUILD_DIR}/elpi" && make && make install )
+
+git_download elpi_hb
+
+( cd "${CI_BUILD_DIR}/elpi_hb" && make && make install )
```

Comment:
One the one hand I would prefer if it was a separate script
On the other hand that would require changing the job name on the gitlab side so maybe not worth it